### PR TITLE
Fix another typo

### PR DESCRIPTION
**Pull request by chu11**
Problem: Some typos were missed in a prior fix.

Fix more typos!

### CODE REVIEW

File: t/t0000-sharness.t [hunk 1]
```diff
@@ -380,12 +380,12 @@ test_expect_success COLOR,PERL_AND_TTY 'sub-sharness still has color' "
 test_expect_success 'EXPENSIVE prereq not activated by default' "
 	run_sub_test_lib_test no-long 'long test' <<-\\EOF &&
 	test_expect_success 'passing test' 'true'
-	test_expect_success EXPENSIVE 'passing suposedly long test' 'true'
+	test_expect_success EXPENSIVE 'passing supposedly long test' 'true'
```
Comment (grondo) [inline t/t0000-sharness.t:383]: `t0000-sharness.t` is from upstream sharness. We should get it fixed there. Note the upstream has moved [here](https://github.com/felipec/sharness)

Comment (chu11) [reply t/t0000-sharness.t:383]: doh!  I knew that!

**Comment by chu11** [comment]
well now this is a singular typo fix.  Perhaps can just leave this here as a catch all PR for stupid little things until its time to do a release?  Similar to the https://github.com/flux-framework/flux-core/pull/5019 PR.

**Comment by chu11** [comment]
i'm going to just close this, absorb it into a PR that solves #1026

**Pull request closed without merging**